acceptance_tests/utilities/mappings.py

```diff
@@ -366,7 +399,7 @@
     'P_IC_ICL1': 'Initial contact letter households - England',
     'P_IC_ICL2B': 'Initial contact letter households - Wales',
     'P_IC_ICL4': 'Initial contact letter households - Northern Ireland',
-    'P_IC_H1': 'Initial contact questionnaire households - England',
+    'P_IC_H1': 'Initial contact question`naire households - England',
```

Comment:
```suggestion
    'P_IC_H1': 'Initial contact questionnaire households - England',
```
Accidental backquote